syft/frameworks/torch/mpc/benchmarks/scripts/benchmark_sigmoid.py

```diff
@@ -0,0 +1,101 @@
+import torch
+import timeit
+import matplotlib.pyplot as plt
+
+from syft.frameworks.torch.mpc.benchmarks.scripts.workers_initialization import workers, hook
+from syft.frameworks.torch.mpc.benchmarks.scripts.benchmark_sample_data import (
+    benchmark_data_sigmoid,
+)
+
+
+def benchmark_sigmoid(method, prec_frac, workers):
+    """
+    This function approximates the sigmoid function using a given method.
+
+    Args:
+        method (str): the name of the method for approximation
+        prec_frac (int): precision value
+        workers (dict): workers used for sharing data
+
+    Returns:
+        diff (int): the difference between the syft and torch approximated value
+
+    """
+    alice, bob, james = workers["alice"], workers["bob"], workers["james"]
+
+    t = torch.tensor([1.23212])
+    t_sh = t.fix_precision(precision_fractional=prec_frac).share(alice, bob, crypto_provider=james)
+    r_sh = t_sh.sigmoid(method=method)
+    r = r_sh.get().float_prec()
+    t = t.sigmoid()
+    # Calculation of the difference between FPT and normal sigmoid (error)
+    diff = (r - t).abs().max()
+    return diff.item()
+
+
+def sigmoid_approximation_plot(benchmark_data_sigmoid):
+    """
+    This function plots the graph for various sigmoidal approximation benchmarks namely
+    'chebyshev', 'maclaurin', 'exp'.
+
+    Args:
+        benchmark_data_sigmoid (list): the sample data to approximate
+
+    Returns:
+        sigmoid_function_approximations_benchmark (png): plotted graph in graph directory
+    """
+
+    # initializing workers
+    worker = workers(hook())
+
+    # initializing graph plot
+    fig, ax1 = plt.subplots()
+    ax2 = ax1.twinx()
+
+    # list for handling graph data
+    x_data = list()
+    y_data = list()
+    y2_data = list()
```

Comment:
I think all these need to be rewritten as literals, since the [tests are failing](https://github.com/OpenMined/PySyft/pull/4031/checks?check_run_id=994680515).  Are you using the [pre-commit package](https://github.com/OpenMined/PySyft/blob/master/CONTRIBUTING.md#setting-up-pre-commit-hook)?

Reply:
done! Thanks :)